.github/workflows/build_and_test.yml

```diff
@@ -127,3 +144,9 @@ jobs:
 
       - name: Build Android Client
         run: npx gulp build --platform=android
+
+      - name: Upload Android Client
```

Comment:
Make the actions fail if no file is found: https://github.com/actions/upload-artifact#customization-if-no-files-are-found